migrations/Version20200421000600.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use App\Helpers\Yaml;
+use App\Helpers\YamlException;
+use App\Helpers\Evaluation\WeightedScoreCalculator;
+
+/**
+ * Manual migration.
+ * Identifies weighted score configs which are in fact uniform and change them accordingly.
+ */
+final class Version20200421000600 extends AbstractMigration
+{
+    private const UNIFORM_ID = 'uniform'; // This should match UniformScoreCalculator::ID
+    private const WEIGHTED_ID = WeightedScoreCalculator::ID; // This should match WeightedScoreCalculator::ID (former SimpleScoreCalculator)
```

Comment:
A bit weird :-) Why `weighted` id is taken from the class and `uniform` not?

Reply:
That was an attempt to minimize dependencies on the rest of the code. WeightedScoreCalculator was necessary, so even the ID was taken from there. But I can fix it if you so desire...

Reply:
No... just curious why...